### PR TITLE
magnite-batch

### PR DESCRIPTION
**Pull request by jakeseawell**
- Replaced image: 'destination-batch-config-auth-filled.png' because of new source field

### CODE REVIEW

File: help/destinations/catalog/advertising/magnite-batch.md [hunk 1]
```diff
@@ -110,7 +110,12 @@ To configure details for the destination, fill in the required and optional fiel
   future.
 *  **[!UICONTROL Description]**: A description that will help you identify this
   destination connection/instance in the future.
-*  **[!UICONTROL Name of your source partner]**: The name you would like to go by as a source in Magnite Streaming's platform
+*  **[!UICONTROL Name of your source partner]**: The name you would like to go by as a source in Magnite Streaming's platform. 
+  
+>[!NOTE]
```
Comment (vfalon) [inline help/destinations/catalog/advertising/magnite-batch.md:115]: Please rephrase this as such:


`The source partner name must be an all-lowercase string which matches the name of the Amazon S3 delivery bucket you have configured with Magnite and set up in the [authenticate to destination](#authenticate) step. The supported characters include 'a-z', '0-9', '-'(dash), or '_'(underscore).`

**Review by vfalon — CHANGES_REQUESTED** [review]
@jakeseawell please see my comment here as well. Thanks!

File: help/destinations/catalog/advertising/magnite-batch.md (hunk 1 above)
Comment (jakeseawell) [reply help/destinations/catalog/advertising/magnite-batch.md:115]: Changed the Note due to the new requirement that allows uppercase letters (that later get converted to lowercase). Please review. Thanks!

File: help/destinations/catalog/advertising/magnite-batch.md [hunk 2]
```diff
@@ -110,7 +110,11 @@ To configure details for the destination, fill in the required and optional fiel
...
+>
+>The source partner name must be a string which matches the name of the Amazon S3 delivery bucket you have configured with Magnite and set up in the [authenticate to destination](#authenticate) step. The supported characters include 'a-z', A-Z, '0-9', '-'(dash), or '_'(underscore). This string is allowed to contain uppercase characters, but it will be converted to all lowercase in order to comply with Magnite's requirements.
```
Comment (vfalon) [inline help/destinations/catalog/advertising/magnite-batch.md]: "This string is allowed to contain uppercase characters, but it will be converted to all lowercase in order to comply with Magnite's requirements."

How is this information useful in any way for a user? If I can enter lowercase or uppercase without getting an error, why would I care what happens behind the scenes? You already have "A-Z" in the supported characters section, so I don't see why we need this sentence.

Also, "A-Z" should be wrapped in 'A-Z' like the other examples.

**Review by vfalon — CHANGES_REQUESTED** [review]
@jakeseawell please see my comment on this update. Thanks!
